index.html

```diff
@@ -22,17 +22,25 @@
 </head>
 
 <body>
-    <header>
+    <header id="nav">
         <nav class="navbar">
             <a href="#" id="logo">My Logo</a>
-            <a href="#" class="menu-icon"><img src="images/Union.png" alt="menu bar"></a>
+            <a href="#" class="menu-icon" id="hbg"><img src="images/Union.png" alt="menu bar"></a>
             <ul class="menu">
-                <li class="menu-items"><a href="#">Portfolio</a></li>
-                <li class="menu-items"><a href="#">About</a></li>
-                <li class="menu-items"><a href="#">Contact</a></li>
+                <li class="menu-items"><a href="#works-section">Portfolio</a></li>
+                <li class="menu-items"><a href="#about">About</a></li>
+                <li class="menu-items"><a href="#contact-me">Contact</a></li>
             </ul>
         </nav>
     </header>
+    <section id="mobile_menu">
+        <img src="images/X.png" alt="menu bar" id="x_btn">
+        <ul id="mobile_ul">
```

Comment:
- [ ] **[Optional] When I scroll after opening the mobile menu, it goes out of view. You may fix this as it will improve the UI**

 `position: fixed` might be the easiest way to address the issue

![scroll](https://user-images.githubusercontent.com/59168430/122969136-f186a900-d3a9-11eb-90a8-625bb4a45e6e.gif)

Reply:
thanks a lot for your review!